app/src/main/java/edu/kmaooad/capstone23/members/controllers/DeleteMemberController.java

```diff
@@ -0,0 +1,49 @@
+package edu.kmaooad.capstone23.members.controllers;
+
+import edu.kmaooad.capstone23.common.CommandHandler;
+import edu.kmaooad.capstone23.common.HandlingError;
+import edu.kmaooad.capstone23.common.Result;
+import edu.kmaooad.capstone23.common.ValidatingHandler;
+import edu.kmaooad.capstone23.members.commands.DeleteMember;
+import edu.kmaooad.capstone23.members.events.DeletedMember;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.DELETE;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import org.eclipse.microprofile.openapi.annotations.media.Content;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponse;
+
+@Path("/members")
+public class DeleteMemberController {
```

Comment:
No need to make separate controller for DELETE. Just use TypicalController with POST of Delete*Something* command.